### DISCUSSION TITLE
string/builder-buffer

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#250] [Announcements]
# string/builder-buffer

在本文中，我们将详细探讨 Java 中 String、StringBuilder 和 StringBuffer 的差异和使用场景。学习如何根据实际需求选择正确的字符串处理类，了解它们在性能、线程安全和内存开销方面的优缺点。本文还将解析它们的内部实现，帮助你更好地理解 Java 字符串处理的原理和最佳实践。

https://javabetter.cn/string/builder-buffer.html

**Comment by 1274580448** [comment]
“我想，关于 String、StringBuilder、StringBuilder 之间的差别，你都搞清楚了吧？”我问。这句话有问题 StringBuffer
